### DISCUSSION TITLE
Is it possible to configure Scheduler HA using oracle 19c?

### DISCUSSION THREAD

**Discussion by dntjr8096** [apache/airflow#17802] [Q&A]
Hi
I'm currently using Oracle 19c.
Can I configure Scheduler HA using Oracle?

According to the airflow document, HA configuration is possible if the DB supports syntax such as 'select ... for update', 'skip locked', 'nowait' and row level locks, but Oracle 19c also supports it.

So, it is poosible to configure Scheduler HA even with Oracle 19c?

**Comment by potiuk** [comment]
Airflow does not currently support Oracle at all. The only officially supported DBs are Postgres and MySQL. Support for MSSQL is coming in 2.2.0 but support for Oracle is not existing. While you might try to do it, it will likely fail and there is no support you will get from the community if it does. 

 See https://github.com/apache/airflow#requirements

**Reply by dntjr8096** [reply]
Thanks for your reply.
I'll use another DB!
